src/Playwright.NUnitTest/Playwright.NUnitTest.csproj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would name it just `Playwright.NUnit`

Reply:
It'll clash with NUnit

Reply:
I don't see these folks are getting any kind of clashes https://github.com/AkkaNetContrib/Akka.TestKit.NUnit

Reply:
is you do `using Akka.TestKit.NUnit` and then say `NUnit.foo` in the code, C# will be confused.